app/scripts/models/notifications.js

```diff
@@ -51,22 +53,20 @@ define([
       });
     },
 
-    profileUpdated: function (data) {
```

Comment:
Very glad you did this. I was tempted to do it before but decided it was out of scope for my stuff.